ui/src/i18n/en.json

```diff
@@ -242,6 +242,10 @@
   "platform.linux.shutting_down_server": "Shutting Down Server",
   "platform.version.restart_required": "Homebridge UI v{{ uiVersion }} has been installed, but the server service is still running v{{ serverVersion }}.",
   "platform.version.service_restart_required": "Restart Required",
+  "platform.terminal.confirm_navigation_title": "Terminal session will be terminated",
+  "platform.terminal.confirm_navigation_message": "Navigating away from this page will terminate your current session, any running processes will be stopped.<br>Are you sure you want to terminate your session?",
```

Comment:
> any running processes will be stopped

would a command not continue to run even though i have exited the terminal (obviously i cannot continue to see the output)?